lists2safebrowsing.py

```diff
@@ -690,8 +690,14 @@ def main():
     if log_file:
         log_file.close()
 
+    # set remote_settings_upload to true for uploading an unversioned list
+    config.set('main', 'remote_settings_upload', 'True')
```

Comment:
I think this line will override the config from the ini file. So, the script will still upload to rs even if the ini file says false. We should only set this to true if the ini file explicitly requests it.

Reply:
I did this to enable remote settings upload just for the unversioned lists. I set the upload to false after the first `publish_to_cloud` call. 
Is there another way for me to set remote settings uploads to True just for the latest Nightly builds?

Reply:
Updated to reflect that we are disabling rs_upload for versioned scripts. The .ini file has to be updated in shavar-prod-lists to set rs_upload to true